### DISCUSSION TITLE
Run gotenberg outside docker container

### DISCUSSION THREAD

**Discussion by Dhopal** [gotenberg/gotenberg#588] [Q&A]
Hi folks, I have the following use case-

On one VM, I can run only one container(don't ask why). Inside this container I'll be running my application which needs to call gotenberg to convert html to pdf. I've thought of running gotenberg binary on localhost inside the container.

Do you think it is a good idea?
Also, is there some doc I can follow to install the dependencies for gotenberg.

Thanks in advance.

**Comment by gulien** [comment]
Hello @Dhopal,

Gotenberg is tightly linked to its container, so I don't recommend installing it outside of it's natural habitat 😅 Why not a second VM with a Gotenberg container inside?

**Reply by Dhopal** [reply]
Second VM with a Gotenberg container inside will increase network cost as well as $ cost. Only, option that I've is to run both inside the same container, and gotenberg on a local host. But, having a hard time figuring out the dependecies and inlcuding it into our base image.
